<launch>
    <include file='$(find gazebo_ros)/launch/empty_world.launch'>
        <arg name="world_name" value="$(env HOME)/gazebo_worlds/modified_playpen/world.world"/>
    </include>
    <include file='$(find husky_gazebo)/launch/spawn_husky.launch'>
        <arg name="z" value="0.10"/>
    </include>
</launch>